### PR TITLE
Beginning to edit READ.me for new repo

### PR DESCRIPTION
**Pull request by michellewong793**
### Description

current behavior: broken read.me since it was ported from the monorepo 

changes: 
- Deleted things from monorepo 
- Added Valora specific copy 
- Left space for instructions on how to build Valora 
- Future work: port over instructions on how to build Valora here

### Other changes

n/a 
### Tested

n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,9 @@
   </a>
 </p>
 
-**Celo Monorepo - Official repository for core projects comprising the Celo platform**
+**Valora Monorepo - Official repository for Valora**
 
-This repository contains the source code for the Celo core projects including the [smart contracts](https://github.com/celo-org/celo-monorepo/tree/master/packages/protocol), [wallet app](https://github.com/celo-org/celo-monorepo/tree/master/packages/mobile),
-and other packages. The source code for the Celo Blockchain which operates a node on the Celo Network is kept in a separate repo [here](https://github.com/celo-org/celo-blockchain).
+This repository contains the source code for Valora, on both iOS and Android.
```
Comment (jmrossy) [inline README.md:9]: Consider putting a sentence or two here about what Valora is.
Something like this: https://github.com/celo-org/celo-monorepo/pull/7600/files#diff-45798d5ec1a1e504b1d5d8ed55ba1e4353449e61b47eeffe1de4e727d1c96bcfR15

File: README.md [hunk 2]
```diff
@@ -43,54 +42,27 @@ Contents:
...
 - [Documentation](#docs)
 - [Issues](#issues)
 - [Repo Structure](#repo)
 - [Contributing](#contributing)
 - [Ask Questions, Find Answers, Get in Touch](#ask)
 - [License](#license)
   <!-- /TOC -->
+  
+## 🥅 <a id="build"></a>How to Build Valora Locally 
+Follow instructions [here](https://github.com/celo-org/celo-monorepo/blob/master/SETUP.md#running-the-mobile-wallet). (This part of the READ.me is in construction! To be rewritten)
```
Comment (jmrossy) [inline README.md:57]: I see you're using the term READ.me here and in this PR. Do you mean README.md? I think that's the more common name for these files.

Comment (jmrossy) [inline README.md:57]: We should move those setup instructions out of the monorepo to here.

**Review by jmrossy — COMMENTED** [review]
A good start! 

This seems like a good opportunity to update the https://github.com/celo-org/wallet/tree/main/packages/mobile too. We should remove or update the old screenshots and ensure the docs are still correct

File: README.md (hunk 1 above)
Comment (michellewong793) [reply README.md:9]: sounds good!

File: README.md (hunk 2 above)
Comment (michellewong793) [reply README.md:57]: oops! yes haha

Comment (michellewong793) [reply README.md:57]: 👍

Comment (michellewong793) [reply README.md]: the instructions are actually here: https://github.com/celo-org/wallet/tree/main/packages/mobile will link to it instead.

File: README.md [hunk 3]
```diff
@@ -106,24 +82,7 @@ See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a
 
 The repository has the following packages (sub projects):
 
-- [attestation-service](packages/attestation-service) - service run by validators on the Celo network to send SMS messages, enabling attestations of user phone numbers and their accounts on the Celo network
-- [blockchain-api](packages/blockchain-api) - service that uses Blockscout to present view of transactions by account for Celo Wallet activity feed
```
Comment (gnardini) [inline README.md:110]: We should probably  leave `blockchain-api` and `notification-service` here.

File: README.md [hunk 4]
```diff
@@ -43,54 +46,27 @@ Contents:
...
 - [Documentation](#docs)
 - [Issues](#issues)
 - [Repo Structure](#repo)
 - [Contributing](#contributing)
 - [Ask Questions, Find Answers, Get in Touch](#ask)
 - [License](#license)
   <!-- /TOC -->
+  
+## 🥅 <a id="build"></a>How to Build Valora Locally 
+To setup Valora locally, follow directions here: https://github.com/celo-org/wallet/tree/main/packages/mobile.
```
Comment (jmrossy) [inline README.md:61]:
```suggestion
To setup Valora locally, follow [setup instructions](https://github.com/celo-org/wallet/tree/main/packages/mobile).
```

File: README.md [hunk 5]
```diff
@@ -43,54 +46,27 @@ Contents:
 
 <!-- TOC -->
 
+- [How to build Valora](#build)
```
Comment (jmrossy) [inline README.md:49]: Recommend moving this section down to after The Valora Stack

File: README.md [hunk 6]
```diff
@@ -43,54 +46,27 @@ Contents:
...
-  <img src="https://storage.googleapis.com/celo-website/docs/network.png" alt="Celo network" width="900" style="border:none;"/>
-  <br />
-  <i>Topology of a Celo Network</i>
-</p>
-
-- **Applications:** Applications for end users built on the Celo platform. The Celo Wallet app, the first of an ecosystem of applications, allows end users to manage accounts and make payments securely and simply by taking advantage of the innovations in the Celo protocol. Applications take the form of external mobile or backend software: they interact with the Celo Blockchain to issue transactions and invoke code that forms the Celo Core Contracts’ API. Third parties can also deploy custom smart contracts that their own applications can invoke, which in turn can leverage Celo Core Contracts. Applications may use centralized cloud services to provide some of their functionality: in the case of the Celo Wallet, push notifications and a transaction activity feed.
+## 🧱 <a id="stack"></a>The Valora Stack
+(TBW) 
 
 ## 📚 <a id="docs"></a>Documentation
```
Comment (jmrossy) [inline README.md:71]: Please merge this section with the build section above. Let's not maintain two separate setup docs :)

**Review by aaronmgdr — APPROVED** [review]
looks good once the changes that have been suggested are made

File: README.md (hunk 5 above)
Comment (michellewong793) [reply README.md:49]: sure!

File: README.md (hunk 6 above)
Comment (michellewong793) [reply README.md]: done

**Pull request merged**